component/portal/src/main/java/org/exoplatform/portal/rest/GroupRestResourcesV1.java

```diff
@@ -42,6 +42,8 @@ public class GroupRestResourcesV1 implements ResourceContainer {
   public static final int     DEFAULT_LIMIT  = 20;
 
   public static final int     DEFAULT_OFFSET = 0;
+  public static final String     SPACES_GROUP_TYPE = ".spaces";
+  public static final String     ROOT_GROUP_TYPE = "root_type";
```

Comment:
```suggestion
  public static final String     ROOT_GROUP_TYPE = "root_type";
```
Why adding the parent group as embedded exclusion ? Shouldn't we be able to search for parent groups ?

Reply:
Functional decision, IMO we don't add members to these groups.